daisy_workflows/export/export_disk.sh

```diff
@@ -57,5 +59,23 @@ TARGET_SIZE_BYTES=$(gsutil ls -l "${GCS_PATH}" | head -n 1 | awk '{print $1}')
 TARGET_SIZE_GB=$(awk "BEGIN {print int(((${TARGET_SIZE_BYTES}-1)/${BYTES_1GB}) + 1)}")
 serialOutputPrefixedKeyValue "GCEExport" "target-size-gb" "${TARGET_SIZE_GB}"
 
+# Generate SBOM if flag is set
+function runSBOMGeneration() {
+  gsutil cp gs://koln-bucket/syft_0.59.0_linux_amd64.tar.gz .
+  tar -xf syft_0.59.0_linux_amd64.tar.gz
```

Comment:
Please update this to an existing bucket name or bucket already in use for compute-image-tools (avoiding a personally named bucket).

Reply:
Not fixed yet, this will be changed to gs://gce-image-build-resources once I have finished promoting from gs://gce-image-build-resources-rtp using the Artifact Releaser command.

Reply:
currently uses a user input path to gcs instead